src/main/java/com/nina/modelling/shoes/KidShoe.java

```diff
@@ -0,0 +1,34 @@
+package com.nina.modelling.shoes;
+
+public class KidShoe extends Shoe {
```

Comment:
This PR should not include code and files from Task1 but only from Task2.
You should create a branch from task 1 branch and create PR to task 1 branch.
After Task 1 will be merged to master we should change this PR to merge to master.